src/fireflyClient.ts

```diff
@@ -1354,4 +1357,36 @@ export class FireflyClient {
     );
     return response;
   };
+
+  /**
+   * Reset timer for cancel on disconnect for open orders
+   * @param params PostTimerAttributes containing the countdowns of all markets
+   * @returns PostTimerResponse containing accepted and failed countdowns. If status is not 201, request wasn't successful.
+   */
+  resetCancelOnDisconnectTimer = async (params: PostTimerAttributes) => {
```

Comment:
If these routes are only available on staging for now let’s put condition on these methods for time being with message “not available on prod” if you’re planning to merge the PR for testing, If not then PR is good to go but let’s not merge it until everything at backend is up

Reply:
Same for all the newly added wrappers